commons/src/main/java/org/eclipse/kapua/commons/service/event/store/internal/EventStoreServiceImpl.java

```diff
@@ -42,6 +42,7 @@
     private static final AuthorizationService AUTHORIZATION_SERVICE = LOCATOR.getService(AuthorizationService.class);
     private static final PermissionFactory PERMISSION_FACTORY = LOCATOR.getFactory(PermissionFactory.class);
 
+    private static final String KAPUA_EVENT_ID = "kapuaEventId";
```

Comment:
For all of this class I would use `scopeId` and `id` for as argument names so we can fall back on values coming from the `EntityAttributes`